doc/installation_windows.md

```diff
@@ -92,3 +89,9 @@ When set, the compiler will check the target WinSDK version, and abort the build
 
 **Note** In case of multiple WinSDK version installed, CMake automatically selects the latest version of SDK. In some cases, however, the automatic selection may fail. Then you need to manually retarget the solution for the proper WinSDK version: 
 ![win_retarget_platform](./img/win_retarget_platform.png)
+
+## Troubleshooting Issues
+
+| Error                                   | Cause                            | Correction Steps                                                                                                            |
+|-----------------------------------------|----------------------------------|-----------------------------------------------------------------------------------------------------------------------------|
+| The camera is not recognized by the SDK | The SDK has not access to camera | Make sure that "Allow desktop apps to access your camera" setting is set to on in Windows OS (Setting -> Privacy -> Camera) |
```

Comment:
`has not access to camera` -> `has no access to the camera` or `does not have access to the camera`